### PR TITLE
update glide.yaml

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by q384566678** [comment]
ping @coolljt0725 @xiekeyang @stevvooe

**Comment by xiekeyang** [comment]
If updating are fixing some problem?

**Comment by q384566678** [comment]
Not currently, just an updated version.

File: glide.lock [hunk 1]
```diff
@@ -1,5 +1,5 @@
-hash: bd44847c5d8eb9a431262aa7dc304c235ed5fd59a5e106b827a4c171e500c670
-updated: 2017-07-20T09:42:45.593735247+08:00
+hash: 8a6d0301e099f3d4860e9a4ae741b65c82e318b4dc969652589df0754f96b6ae
+updated: 2017-06-29T15:09:15.857129041+08:00
```
Comment (stevvooe) [inline glide.lock]: Why is this timestamp from before the current one?

Comment (q384566678) [inline glide.lock]: My PR came up earlier, which led to this situation when `rebase` was done. Now updated.

**Comment by stevvooe** [comment]
LGTM

**Comment by coolljt0725** [comment]
LGTM

**Pull request merged**